src/pages/ServiceList/__tests__/ServiceListComponent.test.ts

```diff
@@ -14,56 +21,65 @@ const makeService = (name: string, errRatio: number): WithServiceHealth<ServiceL
 
 describe('SortField#compare', () => {
   describe('sortField = health, is ascending', () => {
+    beforeAll(() => {
+      setServerConfig(healthConfig);
+    });
     const sortField = ServiceListFilters.sortFields.find(s => s.title === 'Health')!;
     it('should return >0 when A service health is better than B (priority)', () => {
-      const serviceA = makeService('A', 0);
-      const serviceB = makeService('B', 0.2);
+      const serviceA = makeService('A', { http: { '200': 1.6 } }, {});
```

Comment:
These tests would benefit to have a comment with the default tolerance config used.